std/uni.d

```diff
@@ -9108,11 +9108,20 @@ bool isMark(dchar c)
 @safe pure nothrow @nogc
 bool isNumber(dchar c)
 {
-    return numberTrie[c];
+    // optimization
+    if (c < 0xAA)
+    {
+        return c >= 48 && c <= 57;
+    }
+    else
+    {
+        return numberTrie[c];
+    }
 }
 
-@safe unittest
+unittest
 {
+    import std.stdio;
```

Comment:
Is this import used? Also, why was `@safe` removed? Something left over from debugging?